Mentioned IMGUI_USE_WCHAR32

frink opened the pull request:
There is no mention of IMGUI_USE_WCHAR32 in the font documentation.
This stumped me for a while. I'd like to save someone else such frustration.

There may be a better more obvious place for this.
Just wanted to bring attention to the omission of documentation on this feature. Thanks.

ocornut commented:
Hello, thank you!
would be good to clarify in the comments what the “full” range means, fix the typo and add the # in the define.

frink commented:
I've added clarifying wording both to the `docs/Fonts.md` and `imconfig.h` removed the word full in both places per your request and fixed the typo for #define.

ocornut commented:
There are still multiple typos in your comments, including spelling of the word “define”, and “or in your own header before including `imgui.h` in your project” is incorrect.

frink commented:
Fixed the typo and removed the offending or... line.

ocornut commented:
I'll let you find the two other typos :)

frink commented:
Face plant!!! - And that's what you get for doing a PR while playing LEGO with your kids...
